bin/cassandra.in.sh

```diff
@@ -153,7 +153,9 @@ esac
 
 # Read user-defined JVM options from jvm-server.options file
 JVM_OPTS_FILE=$CASSANDRA_CONF/jvm${jvmoptions_variant:--clients}.options
-if [ $JAVA_VERSION -ge 17 ] ; then
+if [ $JAVA_VERSION -ge 22 ] ; then
```

Comment:
Do we know if 21 wants the same as 17 or the same as 22?

Reply:
17 files were pulled from Apache when 21 was tested, so I think people were using the 17 file and kept on adding things there. Now when we add this config for 22, we preserve people to use 17 for anything between 17 and 22 and we are not changing anything there in those files. I would assume from now on people will be using 22+ and they should be pushing things only in the 22 file.